apps/quickstart-integrations-monday-code/.env

```diff
@@ -1,2 +1,3 @@
 MONDAY_SIGNING_SECRET=abcd1234
-PORT=8080
\ No newline at end of file
+PORT=8080
+MNDY_SERVER_ADDRESS=http://localhost:8080
```

Comment:
What is this used for?

Reply:
To run & test it locally, we need to know what is the server local address in order to post the queue item payload